common/autoware_debug_tools/autoware_debug_tools/processing_time_visualizer/node.py

```diff
@@ -0,0 +1,82 @@
+import curses
```

Comment:
Please add README about what is the purpose of this script and how to use it.
You can refer to README of planning_debug_tools in autoware_tools.